static/src/stylesheets/module/facia/_slices.scss

```diff
@@ -239,6 +210,29 @@ Hence why a greater depth of selector specificity is needed.
     }
 }
 
+.fc-slice__popular-mpu {
+    .ad-slot {
+        min-height: 274px;
```

Comment:
Where does 274px come from? I notice _items.scss has the following entry -

    .fc-slice__item--mpu-candidate {
        @include mq(tablet) {
            min-height: 25p + $mpu-ad-label-height;
        }
    }

I don't know if perhaps something could be rationalized between them...